packages/@react-spectrum/menu/src/ActionMenu.tsx

```diff
@@ -24,7 +24,7 @@ import {useMessageFormatter} from '@react-aria/i18n';
 import {useSlotProps} from '@react-spectrum/utils';
 
 function ActionMenu<T extends object>(props: SpectrumActionMenuProps<T>, ref: FocusableRef<HTMLButtonElement>) {
-  props = useSlotProps(props, 'actionmenu');
+  props = useSlotProps(props, 'actionMenu');
```

Comment:
Brings it in line with the slot naming convention set up by ActionGroup, ButtonGroup etc

Reply:
fyi, this was the only reference i found in quarry for the 'actionmenu' slot https://git.corp.adobe.com/frontend/quarry/blob/9aa61397fad5f2b919d90bc67bad3c0fa5f3efbd/packages/%40quarry/inventory/src/views/list/ListViewItem.tsx#L103
you may want to double check me on that and make sure this is still safe

Reply:
Hm, I don't believe this change will affect them, if anything that should already be broken if they are relying on getting our ListViewItem class from the actionMenu slot: https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/list/src/ListViewItem.tsx#L224

Reply:
Oh never mind, that is a `actionmenu` grid area not a slot so the above change doesn't affect it at all. The `actionmenu` matches up with the ListView css grid area: https://github.com/adobe/react-spectrum/blob/844549837a0bfdc5943c8a6214dbd4576d027040/packages/%40react-spectrum/list/src/styles.css#L230-L231